src/section-solving-one-step-equations-involving-percentages.mbx

```diff
@@ -150,7 +193,77 @@
+                                        ymin=0,ymax={},
+                                        grid = none,
+                                        %axis x line = bottom,
+                                        axis y line = left,
+                                        axis line style={->}, % arrows on the axis
+                                        yticklabel style={},
+                                        y label style={at={(axis description cs:-0.1,.5)},rotate=90,anchor=south},
+                                        ybar,
+                                        enlargelimits=0.15,
+                                        legend style={at={(0.5,-0.2)},
+                                          anchor=north,legend columns=-1},
+                                        ylabel={Number of Students},
+                                        xlabel={},
+                                        symbolic x coords={freshman,sophamore,junior,senior},
+                                        xtick=data,
+                                        nodes near coords, 
+                                        nodes near coords align={vertical},
+                                        x tick label style={rotate=45,anchor=east},
+                                        ]
+                                        \addplot coordinates {(freshman,134) (sophamore,96) 
+                                            (junior,86) (senior,103)};
+                                    \end{axis}
+                                \end{tikzpicture}
+                            ]]>
+                        </latex-image-code>
+                    </image>
+                </figure>
+            </statement>
+            <solution>
+                <p>The school's total number of students is:<me>134+96+86+103=419</me></p>
+
+                <p>Now, we can translate the main question:</p>
+
+                <blockquote>
+                    <p><q>What percentage of the school's student population is freshman?</q></p>
+                </blockquote>
+
+                <p>into:</p>
+
+                <blockquote>
+                    <p><q>What percent of <m>419</m> is <m>134</m>?</q></p>
+                </blockquote>
+
+                <p>Using <m>P</m> to represent the unknown quantity, we write and solve the equation:<md>
+                    <mrow>\overbrace{\strut P}^{\text{what percent}}\overbrace{\strut \cdot}^{\text{of}} \overbrace{\strut 419}^{\text{419}}\amp\overbrace{\strut =}^{\text{is}}\overbrace{\strut 134}^{\text{134}}</mrow>
+                    <mrow>\divideunder{P\cdot 419}{419}\amp=\divideunder{134}{419}</mrow>
+                    <mrow>P\amp=0.319809\ldots</mrow>
+                    <mrow>P\amp\approx31.98\%</mrow>
+                </md></p>
+
+                <p>Approximately <m>31.98\%</m> of the school's student population is freshman.</p>
+            </solution>
+        </example>
+
+<!--        
```

Comment:
Since the section is taking on more examples to get graph problems in here, commenting out this one is OK. But there is a paragraph in the solution here about how one dives into a word problem, and I think it should be kept. Just maybe moved somewhere else (and lightly edited) if this example is commented out.